pom.xml

```diff
@@ -1541,9 +1542,7 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-        <pluginManagement>
-            <plugins>
+
```

Comment:
@colings86 after completing nuking the project and starting again with `mvn eclipse:eclipse` and then importing as suggested, it works.

Thanks for the help, and sorry for the noise caused by this PR.

The other warning about duplicate declarations has since been fixed in e60f61d0a9e928bcd1ea6ee965e79f5003f76a95 so I will close this PR.